cmd/nerdctl/rename_windows_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestRename(t *testing.T) {
```

Comment:
Just curious: Why not have common tests in rename_test.go and platform specific tests in rename_<os>_test.go using build tags?

Reply:
As far as I could tell all the existing tests were laid out using the filename suffixes for build constraints so I strived to keep it that way.

I also needed to paraemtrize the test image since `testutils.CommonImage` is only defined on Linux/Windows/FreeBSD and that would prevent the tests from compiling on any other future platform we'd like to support.

We could alternatively set a default `testutils.CommonImage` for all platforms and thus have the `TestRename*` tests be 100% shared across all platforms.
I'd personally rather we move the `CommonImage` constants from `testutils` to here since it's only ever referenced in the `./cmd/nerdctl/...` tests anyway...